examples/quickstart/jupyter-notebooks/druidapi/druidapi/__init__.py

```diff
@@ -13,14 +13,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .druid import DruidClient
+from druidapi.druid import DruidClient
 
 def jupyter_client(endpoint) -> DruidClient:
     '''
     Create a Druid client configured to display results as HTML withing a Jupyter notebook.
     Waits for the cluster to become ready to avoid intermitent problems when using Druid.
     '''
-    from .html import HtmlDisplayClient
+    from .html_table import HtmlDisplayClient
```

Comment:
Didn't I move all the display stuff into the `html.py` file as one of my last commits?

Reply:
Yes -- I updated the filename to avoid a circular import error:
```
ImportError: cannot import name 'escape' from partially initialized module 'html' (most likely due to a circular import)
```
Will re-update `html_table.py` and `text.py` as we discussed.